src/wp-includes/global-styles-and-settings.php

```diff
@@ -255,3 +255,70 @@ function ( $item ) {
 		}
 	}
 }
+
+/**
+ * Whether a theme or its parent have a theme.json file.
+ *
+ * @since 6.2.0
+ *
+ * @return boolean
```

Comment:
```suggestion
 * @return bool
```
Core uses `bool` rather than `boolean`. See [here](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#phpdoc-tags) (beside `@return`).